packages/code-studio/.env.development

```diff
@@ -1,15 +1,10 @@
-REACT_APP_ENABLE_LOG_PROXY=false
-# Proxy set in package.json to handle CORS for most URLs (/notebooks, /jsapi, /layouts, etc)
-# https://create-react-app.dev/docs/proxying-api-requests-in-development/
-# However, we still need to specify the jsapi path because otherwise it doesn't work in Chrome
-# https://github.com/facebook/create-react-app/issues/5280
-REACT_APP_CORE_API_URL=http://localhost:10000/jsapi
+VITE_ENABLE_LOG_PROXY=false
+VITE_CORE_API_URL=http://localhost:10000/jsapi
+VITE_FAVICON=/favicon-cc-app-dev.svg
 
-PUBLIC_URL=/
-REACT_APP_INTERNAL_COMPONENT_PLUGINS=ExamplePlugin
+BASE_URL=/
+VITE_INTERNAL_COMPONENT_PLUGINS=ExamplePlugin
 PORT=4000
 REACT_APP_LOG_LEVEL=4
```

Comment:
I don't think this is needed anymore, but you should define `VITE_LOG_LEVEL`

Reply:
Good catch. Probably missed this in a rebase